crates/fuel-core/src/schema/block.rs

```diff
@@ -292,7 +295,7 @@ impl BlockQuery {
         crate::schema::query_pagination(after, before, first, last, |start, direction| {
             Ok(blocks_query(
                 query.as_ref(),
-                start.map(Into::into),
+                start.map(Into::<BlockHeight>::into).into(),
```

Comment:
It doesn't look like we can avoid the `Option` here, so this is where we are giving the business-level context to a `None` I guess.